react/features/participants-pane/components/web/MeetingParticipantItem.js

```diff
@@ -155,12 +169,41 @@ function MeetingParticipantItem({
     participantActionEllipsisLabel,
     youText
 }: Props) {
+
+    const [ hasAudioLevels, setHasAudioLevel ] = useState(false);
+    const [ registeredEvent, setRegisteredEvent ] = useState(false);
+
+    const _updateAudioLevel = level => {
```

Comment:
This function will be recreated on every render. Maybe it's a good idea to use `useCallback`.